aten/src/ATen/nnapi/nnapi_bind.cpp

```diff
@@ -3,15 +3,18 @@
 #include <ATen/ATen.h>
 #include <torch/custom_class.h>
 
+#include <ATen/nnapi/nnapi_bind.h>
 #include <ATen/nnapi/nnapi_wrapper.h>
 #include <ATen/nnapi/nnapi_model_loader.h>
 
 
 namespace torch {
 namespace nnapi {
-namespace {
+namespace bind {
 
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
 nnapi_wrapper* nnapi;
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
 nnapi_wrapper* check_nnapi;
```

Comment:
These are declared in the header, because the other types declared there (specifically `ModelPtr` and `CompilationPtr`), depend on those global variables.
It's defined in .cpp as well, because if it's directly defined in .h (without `extern`), there are ODR warnings.